ocean-client/addons/command-center/src/command-center.ts

```diff
@@ -9,92 +10,121 @@ import { SetRange } from './commands/set-range'
-                break
-            case Commands.SetStableArbSize:
-                command = new SetStableArbSize(telegram, store, commandData)
-                break
-            case Commands.SetAutoDonation:
-                command = new SetAutoDonation(telegram, store, commandData)
-                break
-            default:
-                console.log("ignore " + message.command)
-                break
-        }
-        await telegram.send("executing " + message.command)
-        await command?.execute()
+const MIN_MAXI_VERSION = { major: '2', minor: '0' }
+const MIN_REINVEST_VERSION = { major: '1', minor: '0' }
+
+async function execute(
+  messages: Message[],
+  telegram: Telegram,
+  store: Store,
+  availableBots: AvailableBots,
+  versionCheck: VersionCheck,
+) {
+  for (const message of messages) {
+    let commandData = message.command.split(' ')
+    if (commandData.length == 0) {
+      continue
+    }
+    let command: Command | undefined
+    switch (commandData[0]) {
+      case Commands.Help:
+        command = new Help(telegram, store, availableBots, commandData)
+        break
+      case Commands.Bots:
+        command = new Bots(telegram, store, availableBots, commandData, versionCheck)
+        break
+      case Commands.Check:
+        command = new Check(telegram, store, availableBots, commandData)
+        break
+      case Commands.Execute:
+        command = new Execute(telegram, store, availableBots, commandData)
+        break
+      case Commands.Skip:
+        command = new Skip(telegram, store, availableBots, commandData)
+        break
+      case Commands.Resume:
+        command = new Resume(telegram, store, availableBots, commandData)
+        break
+      case Commands.RemoveExposure:
+        command = new RemoveExposure(telegram, store, availableBots, commandData)
+        break
+      case Commands.SetRange:
+        command = new SetRange(telegram, store, availableBots, commandData)
+        break
+      case Commands.SetReinvest:
+        command = new SetReinvest(telegram, store, availableBots, commandData)
+        break
+      case Commands.SetStableArbSize:
+        command = new SetStableArbSize(telegram, store, availableBots, commandData)
+        break
+      case Commands.SetAutoDonation:
+        command = new SetAutoDonation(telegram, store, availableBots, commandData)
+        break
+      default:
+        console.log('ignore ' + message.command)
+        break
     }
+    await telegram.send('executing ' + message.command)
+    await command?.execute()
+  }
 }
 
 function isFirstRun(settings: StoredSettings): boolean {
-    return settings.lastExecutedMessageId === 0
+  return settings.lastExecutedMessageId === 0
 }
 
 export async function main(): Promise<Object> {
+  const store = new Store()
+  const settings = await store.fetchSettings()
+
+  const telegram = new Telegram(settings, '\\[CommandCenter ' + process.env.AWS_REGION + ' ' + VERSION + ']')
```

Comment:
I have changed the parse_mode of Telegram API to Markdown. In Markdown `[` would start a link. While testing I have seen that just escaping the beginning of the link is enough to avoid trying to send a link.